tests/conftest.py

```diff
@@ -674,9 +684,31 @@ def factory(
             storageclass (object): ocs_ci.ocs.resources.ocs.OCS instance
                 of 'StorageClass' kind.
             size (int): The requested size for the PVC
-            access_mode (str): ReadWriteOnce, ReadOnlyMany or ReadWriteMany.
-                This decides the access mode to be used for the PVC.
-                ReadWriteOnce is default.
+            access_modes (list): List of access modes. One of the access modes
```

Comment:
Please ensure that all existing calls to this factory, are modified according to this change

Reply:
These are the tests which use mutlt_pvc_factory:
1. test_disruptive_during_pod_pvc_deletion
2. test_disruptive_during_pod_pvc_deletion_and_io
3. test_run_io_multiple_pods

1 and 2 are updated in this PR.
3 does not pass access_mode parameter. So default value of access mode will be taken which is same as before.  So this enhancement will not have any impact on 3.